routers/web/web.go

```diff
@@ -484,9 +484,7 @@ func RegisterRoutes(m *web.Route) {
 		m.Get("/attachments/{uuid}", repo.GetAttachment)
 	}, ignSignIn)
 
-	m.Group("/{username}", func() {
```

Comment:
@zeripath I think we can group up the `"/{username}"` route above with this

Reply:
```go
	reqSignIn := context.Toggle(&context.ToggleOptions{SignInRequired: true})
	ignSignIn := context.Toggle(&context.ToggleOptions{SignInRequired: setting.Service.RequireSignInView})
```
yes should be posible

Reply:
They have different signin checks.

Reply:
well not realy, you could do ignSignIn for the grup and overwrite it for Post ... 

- but I'm not sure what the router can obtimize best